pkg/apis/fleet.cattle.io/v1alpha1/bundle.go

```diff
@@ -191,6 +191,8 @@ type BundleDeploymentOptions struct {
 	ForceSyncGeneration int64             `json:"forceSyncGeneration,omitempty"`
 	YAML                *YAMLOptions      `json:"yaml,omitempty"`
 	Diff                *DiffOptions      `json:"diff,omitempty"`
+	Resync              bool              `json:"resync,omitempty"`
```

Comment:
Do we still need `Resync`? Or we can check if `ResyncPolicy != nil` to decide whether user want to resync or not?

Reply:
I see the point that nick made. It should be fine.